job_definitions/release_pipeline.yml

```diff
@@ -34,18 +34,16 @@
               }
             } catch(error) {
               echo "Functional tests failed: ${error}"
-              timeout(time:1, unit:'DAYS') {
-                script {
-                  env.FAILED_TEST_ACTION = input message: 'Functional tests failed. Retry, skip or abort?',
-                    parameters: [
-                      [
-                        $class: 'ChoiceParameterDefinition',
-                        choices: 'Retry\nSkip',
-                        description: 'Failed functional tests action',
-                        name: 'action'
-                      ]
+              script {
+                env.FAILED_TEST_ACTION = input message: 'Functional tests failed. Retry, skip or abort?',
+                  parameters: [
+                    [
+                      $class: 'ChoiceParameterDefinition',
+                      choices: 'Retry\nSkip',
+                      description: 'Failed functional tests action',
+                      name: 'action'
                     ]
-                }
+                  ]
                 echo "Action chosen was ${env.FAILED_TEST_ACTION}"
```

Comment:
Not sure if intentional or not, but these `echo`s are now inside the script block that sets the env variable. They weren't before. I don't know what the effect of this is, maybe nothing, thought it worth checking though.

Reply:
👍 fixed